pulsar-common/src/main/java/org/apache/pulsar/common/util/RedactedRole.java

```diff
@@ -0,0 +1,9 @@
+package org.apache.pulsar.common.util;
+
+public final class RedactedRole {
+   private static final String REDACTED = "[REDACTED]";
```

Comment:
Just wondering if it would be fine to log a hash of the role so that there could be some way to track if the same auth role appears in logging. Obviously that depends on what the requirements are for this feature.

Reply:
Having a hash can do the job yes. Which hash function do you want me to apply? As we use token-based authN/authZ, the role contains the token and logs so much on our usage.

Reply:
Used SHA-256.